src/components/content.js

```diff
@@ -249,15 +249,28 @@ class Content extends React.Component {
    * @return {Boolean}
    */
 
+  isEditableInEditor = (target) => {
+    // COMPAT: Text nodes don't have `isContentEditable` property. So, when
+    // `target` is a text node use its parent node for check.
+    const el = target.nodeType === 3 ? target.parentNode : target
+    return (el.isContentEditable) && this.isInEditor(target)
+  }
+
+
+  /**
+   * Check if an event `target` is fired within the editor,
+   * including void nodes and other nested Slate editors.
+   *
+   * @param {Element} target
+   * @return {Boolean}
+   */
+
   isInEditor = (target) => {
     const { element } = this
     // COMPAT: Text nodes don't have `isContentEditable` property. So, when
     // `target` is a text node use its parent node for check.
     const el = target.nodeType === 3 ? target.parentNode : target
```

Comment:
I think we no longer need this line (and comment) in the `isInEditor` function, judging by the comments text since it was only to allow for using `el.isContentEditable`.

Reply:
fixed!